### ISSUE TITLE
r-ladies salzburg in chapter table

### ISSUE THREAD

**Issue by shannonpileggi** [rladies/website#337]
r-ladies salzburg shows incorrectly in chapter table

![image](https://github.com/rladies/rladies.github.io/assets/17747575/e26a6e27-a077-4e56-bdb9-9e75ec98211e)

**Comment by cosimameyer** [comment]
Thanks for bringing it up! Just to clarify: is the first entry (where it shows "NA") R-Ladies Salzburg?

**Comment by shannonpileggi** [comment]
yes! i think when you click on the meetup icon it takes you to the salzburg page.

**Comment by drmowinckels** [comment]
Right! A couple things going on here. Obviously Austria not being capitalised in one, the other being that for some reason we are not getting updated information from meetup. I dont know why. The scripts in meetup_archive are working just fine, but for some reason, newer chapters are not being retrieved.

I've made a small workaround for now, that at least names the chapter properly in the list. But keep this open a little while we I confer regarding the meetup archive functions.
